### PR TITLE
changing delete_container operation to ssh flow

### PR DESCRIPTION
**Pull request by raskin-aleksandr**
adding system test

### CODE REVIEW

**Comment by raskin-aleksandr** [comment]
@orius123 can you take a look

File: content/io/cloudslang/docker/containers/delete_container.sl [hunk 1]
```diff
@@ -31,39 +31,79 @@
...
+          - standard_err
+          - exception
+        navigate:
+          SUCCESS: SUCCESS
+          FAILURE: FAILURE
+          FAIL_VALIDATE_SSH: FAILURE
   outputs:
-    - container_ID: returnResult
-    - error_message: "'' if 'STDERR' not in locals() else STDERR if returnCode == '0' else returnResult"
+    - result
```
Comment (orius123) [inline content/io/cloudslang/docker/containers/delete_container.sl]: outputs doesn't match flow docs.

File: test/io/cloudslang/docker/containers/test_delete_container.sl [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            - password
+            - container_name
+            - image_name: container_name
+        navigate:
+          SUCCESS: stop_contaier
+          FAILURE: FAIL_RUN_IMAGE
+
+    - stop_contaier:
+        do:
+          images.get_used_images:
```
Comment (orius123) [inline test/io/cloudslang/docker/containers/test_delete_container.sl]: does this op stop the container? `get_used_images`?

**Comment by raskin-aleksandr** [comment]
@orius123 can you take and look. I need this operarion to finish `clear_container`

**Pull request merged**